lib/realtime/postgres_cdc.ex

```diff
@@ -78,9 +78,15 @@ defmodule Realtime.PostgresCdc do
     end
   end
 
-  @spec region_nodes(String.t()) :: [{pid, any}]
+  @doc """
+  Lists the nodes in a region. Sorts by node name in case the list order
+  is unstable.
+  """
+
+  @spec region_nodes(String.t()) :: [{pid(), [node: atom()]}]
   def region_nodes(region) when is_binary(region) do
     :syn.members(RegionNodes, region)
+    |> Enum.sort_by(fn {_pid, [node: node]} -> node end)
```

Comment:
Should we update the current function to return only a list of nodes? Pid inside seems useless

Reply:
Sure